sass/partials/layout/_constrain.scss

```diff
@@ -1,15 +1,25 @@
 // @file
 // Styles for constraining the page width.
 
-.l-constrain {
+@mixin constrain($width: em($max-page-width), $padding: true) {
```

Comment:
I like the idea of adding a mixin for the constrain styles with optional side padding. I think this should go in global/helpers/_constrain.scss instead of in this file though.